standup/status/views.py

```diff
@@ -261,3 +261,12 @@ def csp_violation_capture(request):
                           data=data)
 
     return HttpResponse('Captured CSP violation, thanks for reporting.')
+
+
+def robots_txt(request):
+    if settings.ROBOTS_ALLOW:
+        content = ''
+    else:
+        content = 'User-agent: *\nDisallow: /'
+
+    return HttpResponse(content, content_type='text/plain')
```

Comment:
We could do that... but I think we could also just create an issue for "let's cache things that make sense to cache" and push it off until then.

Either way is fine with me.